Tests/Marketplace/search_and_install_packs.py

```diff
@@ -303,10 +311,19 @@ def install_packs_private(client: demisto_client,
                                  test_pack_path=test_pack_path)
 
 
+def get_error_ids(body: str) -> set[str] | None:
```

Comment:
No need to annotate is as a static method?

Reply:
it's not within a class